src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstance.cs

```diff
@@ -70,12 +72,20 @@ public class VisualStudioInstance
         /// </summary>
         public string InstallationPath { get; }
 
-        public VisualStudioInstance(Process hostProcess, DTE dte, ImmutableHashSet<string> supportedPackageIds, string installationPath)
+        public string DumpDirectoryPath { get; }
+
+        private bool _exceptionHappened;
+
+        public VisualStudioInstance(Process hostProcess, DTE dte, ImmutableHashSet<string> supportedPackageIds, string installationPath, string dumpDirectoryPath)
         {
             HostProcess = hostProcess;
             Dte = dte;
             SupportedPackageIds = supportedPackageIds;
             InstallationPath = installationPath;
+            DumpDirectoryPath = dumpDirectoryPath;
+
+            _exceptionHappened = false;
```

Comment:
simplified

---
In reply to: [121156376](https://github.com/dotnet/roslyn/pull/20129#discussion_r121156376) [](ancestors = 121156376)